src/constants.js

```diff
@@ -363,6 +363,7 @@ export const SKILLS_STATUS_INVOKED = 'skills_invoked_status';
 
 /* ------------------ File Extensions ---------------------- */
 export const FILE_EXTENSION_BOX_NOTE = 'boxnote';
+export const FILE_EXTENSION_BOX_CANVAS = 'boxcanvas';
 export const FILE_EXTENSION_GOOGLE_DOC = 'gdoc';
 export const FILE_EXTENSION_GOOGLE_SHEET = 'gsheet';
```

Comment:
nit: some of the insertions in this PR can be added alphabetically. like here and in `src/utils/file.js`. not a big deal but can help keep things clean